src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -321,8 +354,19 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, $
                 $builder->setSubLocality($values->long_name);
                 break;
 
+            case 'street_address':
+            case 'intersection':
+            case 'political':
+            case 'colloquial_area':
+            case 'ward':
+            case 'neighborhood':
+            case 'premise':
             case 'subpremise':
-                $builder->setValue('subpremise', $values->long_name);
+            case 'natural_feature':
+            case 'airport':
+            case 'park':
+            case 'point_of_interest':
+                $builder->setValue((strpos($type, '_')) ? lcfirst(str_replace('_', '', ucwords($type, '_'))) : $type, $values->long_name);
```

Comment:
Why do we need to transform it from snake_case to camelCase?

Couldn't we just do: 

```php
default:
  if (isset($values->long_name)) {
      $builder->setValue($type, $values->long_name);
  }
  break;
```